test/db/rzil/v810

```diff
@@ -0,0 +1,16 @@
+NAME=Testing the decryption in emulateme
+FILE=bins/v810/emulateme.v810
+ARGS=-a v810
```

Comment:
This file is ELF, thus it should contain the information about the architecture in its header. If Rizin doesn't detect, please extend the code in `librz/bin/formats/elf/` to recognize this architecture automatically.